src/styles/contact-us.scss

```diff
@@ -0,0 +1,122 @@
+.contact-us {
+  display: flex;
+  flex-direction: column;
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    row-gap: 16px;
+
+    @include on-desktop {
+      --columns: 12;
+
+      display: grid;
+
+      gap: 16px 24px;
+      grid-template-columns: repeat(12, 1fr);
+      justify-content: space-between;
+    }
+  }
+
+  &__to-fill-in {
+    display: flex;
+    height: 48px;
+    width: 100%;
+    box-sizing: border-box;
+    border-style: hidden;
+    padding: 14px 16px;
+    border-radius: 8px;
+    background: #f9f9f9;
+    color: #1b2129;
+
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 20px;
+    letter-spacing: 0.15px;
+
+    &--message {
+      height: 148px;
+      font-family: Manrope, sans-serif;
+    }
+
+    @include on-desktop {
+      grid-column: 1 / 7;
+    }
+
+    &::placeholder {
+      font-size: 14px;
+      font-weight: 400;
+      line-height: 20px;
+      letter-spacing: 0.15px;
+      color: #7e7e83;
+    }
+
+    &:hover {
+      cursor: pointer;
+      border: 1px solid #dbdbdb;
```

Comment:
generally for border transition, only change the border color (from transparent to the color that you want), because if you don't do this the content will be carried to the side